core-java-modules/core-java-jndi/src/test/java/com/baeldung/jndi/AJndiUnitTest.java

```diff
@@ -14,7 +14,7 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-class JndiUnitTest {
+class AJndiUnitTest {
```

Comment:
do we need this rename change?
and the comment below?

Reply:
I fixed these 2 points and retested the PR successfully.
I guess all is fine, we can merge this.